game/src/devtools/fifteen_min/mod.rs

```diff
@@ -68,14 +73,43 @@ impl State<App> for Viewer {
 
         if ctx.redo_mouseover() {
             self.hovering_on_bldg = match app.mouseover_unzoomed_buildings(ctx) {
-                Some(ID::Building(id)) => Some(HoverOnBuilding {
-                    id,
-                    tooltip: if let Some(time) = self.isochrone.time_to_reach_building.get(&id) {
-                        Text::from(Line(format!("{} away", time)))
-                    } else {
-                        Text::from(Line("This is more than 15 minutes away"))
-                    },
-                }),
+                Some(ID::Building(hover_id)) => {
```

Comment:
Woops, pre-existing problem that pathfinding makes obvious -- we need to avoid recalculating when the mouse has budged slightly, but we're still on the same building. Quick hack might be something like `if self.hovering_on_bldg.as_ref().map(|hover| hover.id == id) { self.hovering_on_bldg.take() } else { other code }`

And the conversation today about a higher level "moused over something new" event is feeling more and more relevant :)